src/Utils/ErrorMessageBuilder.ts

```diff
@@ -0,0 +1,22 @@
+import { isAxiosError } from 'axios';
+
+export class ErrorMessageBuilder {
+  public static buildMessage(
```

Comment:
The builder pattern cannot be static by nature. This is a static factory, so the names should be corrected accordingly.
```suggestion
  public static genericCommandError(
```